rasa_core/training/interactive.py

```diff
@@ -1317,7 +1317,7 @@ def visualisation_png():
             abort(404)
 
 
-def run_interactive_learning(agent: Agent,
+def run_interactive_learning(agent: 'Agent',
```

Comment:
This shouldn't be a string. The only valid location for a type annotation to be a string is when the method is defined in the same module before the class:
```
def f(a: 'A') -> None:
   pass

class A:
  pass
```
Everywhere else, the type annotation should be the classname. If you want to prevent it from being imported on runtime, put it into an `if typing.TYPE_CHECKING:` at the top
```suggestion
def run_interactive_learning(agent: Agent,
```

Reply:
i think the problem was that the command line interface somehow had the type checking activated